deterministic-build/README.md

```diff
@@ -0,0 +1,49 @@
+# Determnistic Build
+
+This is a docker image to produce reproducible builds of dusk smart contracts.
+
+Usage:
```

Comment:
You immediately start with `docker run` here, but it might also be helpful to temporarily add how you can build the image. 

Ideally we would even have the image published, but we can do that as a follow-up issue.